src/services/accounts.ts

```diff
@@ -42,3 +42,12 @@ export async function getTransactionSigners(server: Server, transaction: Transac
 
   return signers.filter(kp => transactionSignatures.some(sig => kp.verify(transaction.hash(), sig)))
 }
+
+export function getInactiveAccounts(
```

Comment:
this function is doing something useful, but I feel like the fact that it takes in type `WalletAccount` and returns type `Contact` isn't obvious from the name `getInactiveAccounts`. Maybe it should just return the inactive accounts, and the mapping into Contacts could be done separately?

Reply:
renamed it